server/routes/api/articlesRoute.js

```diff
@@ -0,0 +1,54 @@
+import { Router } from 'express';
+import articleController from '../../controllers/articlesController';
+import JWTAuthentication from '../../middlewares/JWTAuthentication';
+import uuidValidator from '../../middlewares/validations/uuidValidator';
+import articleValidation from '../../middlewares/articleValidation';
+
+const router = Router();
+
+/**
+ * @description - Route is use to create an article
+ * @returns - It returns an article object
+ */
+router.post('/article',
+  [JWTAuthentication.authenticateUser,
```

Comment:
For the sake of consistency, could we remove this Array and pass the middlewares normally as you did for the rest.